origin-token-transfer/server/migrations/20181018102506-create-grant.js

```diff
@@ -2,7 +2,7 @@
 
 module.exports = {
   up: async (queryInterface, Sequelize) => {
-    await queryInterface.createTable('Grants', {
+    await queryInterface.createTable('Grant', {
```

Comment:
super nit: in the rest of the stack, we have use all lower case for table name. e.g. "grants" vs "Grants". Same comment for the events table below.